packages/strapi-plugin-upload/admin/src/components/InputMedia/InputFilePreview.js

```diff
@@ -6,7 +6,7 @@ import Flex from '../Flex';
 import Chevron from './Chevron';
 
 const InputFilePreview = ({ file, onClick, isSlider }) => {
-  const fileUrl = file.url.startsWith('/') ? `${strapi.backendURL}${file.url}` : file.url;
+  const fileUrl = file && file.url.startsWith('/') ? `${strapi.backendURL}${file.url}` : file.url;
```

Comment:
Can you update your code when #5624 is merged so we avoid duplicate code?
Furthermore for the preview it would be great to use `formats.thumbnail.url`  when available so we don't have to load the full image, when we have a slow connection the difference is really noticeable.

```
import {prefixFileUrlWithBackendUrl} from 'strapi-helper-plugin`;

const fileUrl = prefixFileUrlWithBackendUrl(get(image, ['formats', 'thumbnail', 'url'], file.url));
```